packages/ember-glimmer/lib/utils/string.ts

```diff
@@ -98,6 +99,19 @@ export function htmlSafe(str: string) {
   return new SafeString(str);
 }
 
+export function deprecatedHTMLSafe(str) {
+  deprecate(
+    'Ember.String namespace is deprecated. Please, import `htmlSafe` from `ember/template`.',
+    false,
+    {
+      id: 'ember-glimmer.ember-string-html-safe',
+      until: '3.5.0',
+      url: ''
```

Comment:
Do we know the URL's for these deprecations? I vaguely recall that there was already an open website PR for them...

Reply:
Adding the URL from https://github.com/emberjs/website/pull/2995/files